templates/frontend/objects/article_details.tpl

```diff
@@ -227,13 +227,17 @@
 				{/foreach}
 
 				{* Article Subject *}
-				{if $article->getLocalizedSubject()}
+				{if !empty($keywords[$currentLocale])}
 					<div class="panel panel-default subject">
 						<div class="panel-heading">
 							{translate key="article.subject"}
 						</div>
 						<div class="panel-body">
-							{$article->getLocalizedSubject()|escape}
+							{foreach from=$keywords item=keyword}
+                						{foreach name=keywords from=$keyword item=keywordItem}
+                  						{$keywordItem|escape}{if !$smarty.foreach.keywords.last}, {/if}
+                						{/foreach}
+              						{/foreach}
```

Comment:
Looks like these changes made it into your pull request because you're working directly from your `master` branch for both changes.

The best way to handle this is to use separate branches when making your changes. That will also prevent your master branch from getting out of sync with mine.

For example, typically a workflow will look like this:

```
# check out the master branch
git checkout master

# sync your master branch with mine
git pull upstream master

# check out a NEW branch for your fix
git checkout -b my_custom_branch

# do all of your changes and then commit and push
git add .
git commit -m "My changes"
git push
```

Then you can issue a pull request from your branch, and you won't get all of your different changes mixed up.